src/Extensions/Hosting.Services.Web/Microsoft.Omex.Extensions.Hosting.Services.Web.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreVersion)</TargetFrameworks>
+    <PlatformTarget>x64</PlatformTarget>
```

Comment:
Some other projects don't have this. Should we have it everywhere or nowhere?

Reply:
It's a requirement of Service Fabric NuGet packages since they are using native code and build for x64, I would really like to have everything using AnyCpu (default value), but packages that depend on SF can only use x64 :(